policyengine_us/parameters/gov/states/or/tax/income/credits/kicker/percent.yaml

```diff
@@ -6,7 +6,8 @@ values:
 metadata:
   unit: /1
   period: year  
-  label: Oregon kicker rate  
+  label: Oregon kicker rate
+  uprating: gov.irs.uprating  
```

Comment:
I dont think this is uprated with inflation -> not every parameter should be please check the legal code for the relevant sections